src/elements/content-sidebar/__tests__/SidebarNavButton.test.js

```diff
@@ -52,4 +52,18 @@ describe('elements/content-sidebar/SidebarNavButton', () => {
 
         expect(button.hasClass('bcs-is-selected')).toBe(expected);
     });
+
+    test('should call onClick with sidebarView when clicked', () => {
+        const mockOnClick = jest.fn();
```

Comment:
Can we implement this test with RTL instead? Any new unit test with Enzyme API adds additional burden to migrating out from it.